tools/mmp/driver.cs

```diff
@@ -1237,6 +1237,9 @@ static int Compile ()
 			if (IsUnified && !arch_set)
 				arch = "x86_64";
 
+			if (arch != "x86_64")
+				ErrorHelper.Warning (134, "32-bit applications should be migrated to 64-bit as they will not run beyond High Sierra \"without compromises\".");
```

Comment:
just a friendly reminder to update this one according to Sebastien's comments above :)